crates/bitwarden-send/src/send.rs

```diff
@@ -361,13 +361,24 @@ impl TryFrom<SendTextModel> for SendText {
 
 #[cfg(test)]
 mod tests {
-    use bitwarden_crypto::{KeyDecryptable, KeyEncryptable, MasterKey};
+    use std::collections::HashMap;
 
-    use super::{Send, SendText, SendTextView, SendType};
-    use crate::{
-        client::{encryption_settings::EncryptionSettings, Kdf},
-        tool::SendView,
-    };
+    use bitwarden_crypto::{Kdf, KeyContainer, KeyDecryptable, KeyEncryptable, MasterKey};
+
+    use super::*;
+
+    struct MockKeyContainer(HashMap<Option<Uuid>, SymmetricCryptoKey>);
+    impl MockKeyContainer {
+        fn new(master_key: MasterKey, user_key: EncString) -> Result<Self, CryptoError> {
+            let user_key = master_key.decrypt_user_key(user_key)?;
+            Ok(Self(HashMap::from([(None, user_key)])))
+        }
+    }
+    impl KeyContainer for MockKeyContainer {
+        fn get_key<'a>(&'a self, org_id: &Option<Uuid>) -> Option<&'a SymmetricCryptoKey> {
+            self.0.get(org_id)
+        }
+    }
```

Comment:
I think long term we should consider extracting this into a test crate or something.

Reply:
Yeah  I think a separate test crate, or maybe even just a dedicated test module in `bitwarden-core` would be helpful in the future